frameworks/Erlang/chicagoboss/rebar.config

```diff
@@ -0,0 +1,7 @@
+{deps, [
```

Comment:
Adding the comment here, but can you rebase your pull request to NOT have unneeded binary files like `favicon.ico` and `chicago-boss.png`? The project size is already rather large without these sorts of unnecessary things.